glue/viewers/profile/layer_artist.py

```diff
@@ -79,6 +79,7 @@ def _calculate_profile_postthread(self):
 
         # Update the data values.
         if len(x) > 0:
+            x = np.arange(len(x))
```

Comment:
Revert?

Reply:
Sure

Reply:
I tried reverting this, but then would get an error that no plot would show in the 1D Profile viewer.

Reply:
Like so: 

<img width="1440" alt="Screenshot 2020-08-18 at 6 31 32 AM" src="https://user-images.githubusercontent.com/26264600/90450565-a1e9a800-e11c-11ea-843d-ec89c0d2afdd.png">

Reply:
I think I can see why this is needed, as WCSAxes plots against array index, but this feels like the wrong place to put it?

Reply:
I guess the question is why x needs updating? why is it not correct when returned from visible_data? Changing it here seems dangerous because there's no guarantee there aren't gaps in the data.

Reply:
This question is still relevant

Reply:
Let me check it again more closely

Reply:
Found out why now... It's because the `x_att` and `x_att_pixel` have been mixed up. Am patching this now.